packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCS/SampleLibraryCS.csproj

```diff
@@ -113,7 +114,6 @@
     <Compile Include="CustomUserControlCS.cs" />
     <Compile Include="CustomUserControlViewManagerCS.cs" />
     <Compile Include="CircleViewManagerCS.cs" />
-    <Compile Include="ReactPackageProvider.cs" />
```

Comment:
>ReactPackageProvider [](start = 22, length = 20)

What if developers want to provide their own package provider? Is it still allowed, or code gen prevents it?

Reply:
They can add the msbuild property `<ReactNativeCodeGenEnabled>false</ReactNativeCodeGenEnabled>` to disable it, per description above.

Reply:
Was also thinking of adding an extra callout to a partial method in the generated code where folks could override if they wanted.